examples/DGmethods/ex_002_IOstrings.jl

```diff
@@ -91,14 +91,14 @@ function Initialise_DYCOMS!(state::Vars, aux::Vars, (x,y,z), t)
     θ_liq = FT(297.5) + (xvert - zi)^(FT(1/3))
     q_tot = FT(1.5e-3)
   end
-    
+
   # Calculate PhasePartition object for vertical domain extent
-  q_pt  = PhasePartition(q_tot, q_liq, q_ice) 
+  q_pt  = PhasePartition(q_tot, q_liq, q_ice)
   #Pressure
   H     = Rm_sfc * T_sfc / grav;
   p     = P_sfc * exp(-xvert/H);
   #Density, Temperature
-  TS    = LiquidIcePotTempSHumEquil_no_ρ(θ_liq, q_pt, p)
+  TS    = LiquidIcePotTempSHumNonEquil(θ_liq, q_pt, p)
```

Comment:
Might be good to make the default argument here density too, as in other functions, and call the version with pressure as argument `_given_pressure` or something like it.